crates/sui-types/src/messages.rs

```diff
@@ -1295,6 +1295,10 @@ impl TransactionKind {
         )
     }
 
+    pub fn to_bytes(&self) -> Base64 {
+        Base64::from_bytes(bcs::to_bytes(self).unwrap().as_slice())
```

Comment:
We should not use `unwrap` here as this will cause the system to panic if `to_bytes` fails.